blog-service/2024-04-20-awso.md

```diff
@@ -0,0 +1,21 @@
+---
+title: AWS Observability 2.8.0 (Observability)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - AWSO
+  - observability
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We are excited to announce the AWS Observability 2.8.0. This update includes: 
+
+- Updates for Amazon RDS - new out-of-the-box dashboards for PostgreSQL based on logs.
+- Redesigned Amazon API Gateway with new metrics and log-based dashboards for Amazon API Gateway access logs for REST, HTTP, and WebSocket API types.
+- Added 20+ new out-of-the-box monitors.
+- Improvements in AWS Observability deployment time.
+- Redesigned apps and monitors aligned with the new Next-Gen Threat Intelligence solution.
```

Comment:
I removed that item because we haven't released the threat intelligence solution yet.

Reply:
AWSO 2.8.0 release will happen after the release of threat intelligence solution. Since there is delay in the release of threat intelligence solution AWSO release is postponed for now.